subsys/bluetooth/host/att.c

```diff
@@ -3087,8 +3087,6 @@ static void bt_att_connected(struct bt_l2cap_chan *chan)
 
 	atomic_set_bit(att_chan->flags, ATT_CONNECTED);
 
-	att_chan_mtu_updated(att_chan);
-
```

Comment:
I don't think this will work for EATT bearers, as the MTU is negotiated as par of the connection request, so if I understood correctly, this will not provide MTU information to the application on EATT connection established. We should probably extend one of the EATT babblesim tests to also care about the MTU exchange to ensure that the change we do here, works as expected for EATT as well.